BUILDING

```diff
@@ -545,9 +539,53 @@ Finally, configure the Emscripten build to use "demo.boot":
  make
 
 
-CROSS-COMPILING SCHEME PROGRAMS
+CROSS COMPILING
+
+To compile Chez Scheme for a target platform that is different than
+the host platform, supply the `--cross` flag to the "configure" script
+while also specifying the target machine type, compiler settings to
+build for the target platform, and `CC_FOR_BUILD=<compiler>` to
+provide a <compiler> suitable for building `bin/zuo`.
+
+For example, to cross compile for Windows with a typical MinGW
+installation, use `--toolprefix=x86_64-w64-mingw32-` to specify tools
+that compile and link via MinGW:
+
+ ./configure --cross -m=ta6nt --toolprefix=x86_64-w64-mingw32- CC_FOR_BUILD=cc
+
+The configure arguments that are needed to cross compile will vary
+with the host and target machines. On a machine where ta6osx is the
+native machine type, for example, cross compiling for tarm64osx may be
+as simple as specifying an `-arch` flag for the C compiler:
+
+ ./configure --cross -m=tarm64osx CFLAGS="-arch arm64" CC_FOR_BUILD=cc
```

Comment:
That would fall into the "If more configuration is needed" case described in the next paragraph. In retrospect, though, that approach is somewhat buried in the explanation, so I'll try revising.

I experimented with ways of passing flags through for the host build, so that you'd be able to specify the host machine type or other configuration options. But they didn't turn out any better than directly using `configure` to build for the host platform (before `configure` for the target platform).

Can you say more about when autodetection goes wrong?